cmd/cli/main.go

```diff
@@ -173,12 +174,20 @@ func main() {
 		os.Exit(1)
 	}
 
+	var flinkAPITimeoutSeconds = int64(10)
```

Comment:
why are you assigning this var first? Assignment is done on line 179 so you should be able to omit this?

Reply:
This was the default value, plan is to have this env variable optional and default to 10 seconds.
I slightly changed implementation as it would error when the env variable was not set